docs/_articles/en/faq.md

```diff
@@ -56,4 +56,23 @@ A: You must:
 #### Q: How can I use `Salesforce CLI Scanner` in my CI/CD?
 A: You can use the `sfdx scanner:run` command in any scripts used by your CI/CD. You'll also probably want to do the following:
 - Use the `-o/--outfile` flag to write your results to a file, so you'll have an artifact of the results.
-- Use the `-v/--violations-cause-error` flag so violations cause a non-zero exit code, since many CI/CD frameworks care about such things.
+- Use the `-s/--severity-threshold` flag so violations above a certain threshold would cause a non-zero exit code, since many CI/CD frameworks care about such things.
```

Comment:
We currently check for `greater than or equal to` the given threshold. Should we use that instead of `above` which seems to imply only `greater than`?

Reply:
yes, makes sense. Will change it.